sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3582,6 +3582,163 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
+            format,
+            df.where('id <=> v.toInt),
+            Array(sources.IsNotNull("id"), sources.EqualTo("id", v)))
+          checkPushedFilters(
+            format,
+            df.where('id <= v.toInt),
+            Array(sources.IsNotNull("id")))
+          checkPushedFilters(
+            format,
+            df.where('id < v.toInt),
+            Array(sources.IsNotNull("id"), sources.Not(sources.EqualTo("id", v))))
+
+          // cases when value > MAX
+          var v1: Int = Short.MaxValue.toInt + 100
+          checkPushedFilters(
+            format,
+            df.where('id > v1),
+            Array(),
+            noScan = true)
+          checkPushedFilters(
+            format,
+            df.where('id >= v1),
+            Array(),
+            noScan = true)
+          checkPushedFilters(
+            format,
+            df.where('id === v1),
+            Array(),
+            noScan = true)
+          checkPushedFilters(
+            format,
+            df.where('id <=> v1),
+            Array(),
+            noScan = true)
+          checkPushedFilters(
+            format,
+            df.where('id <= v1),
+            Array(sources.IsNotNull("id")))
+          checkPushedFilters(
+            format,
+            df.where('id < v1),
+            Array(sources.IsNotNull("id")))
+
+          // cases when value = MIN
+          v = Short.MinValue
+          checkPushedFilters(
+            format,
+            df.where(lit(v.toInt) < 'id),
+            Array(sources.IsNotNull("id"), sources.Not(sources.EqualTo("id", v))))
+          checkPushedFilters(
+            format,
+            df.where(lit(v.toInt) <= 'id),
+            Array(sources.IsNotNull("id")))
+          checkPushedFilters(
+            format,
+            df.where(lit(v.toInt) === 'id),
+            Array(sources.IsNotNull("id"), sources.EqualTo("id", v)))
+          checkPushedFilters(
+            format,
+            df.where(lit(v.toInt) >= 'id),
+            Array(sources.IsNotNull("id"), sources.EqualTo("id", v)))
+          checkPushedFilters(
+            format,
+            df.where(lit(v.toInt) > 'id),
+            Array(),
+            noScan = true)
+
+          // cases when value < MIN
+          v1 = Short.MinValue.toInt - 100
+          checkPushedFilters(
+            format,
+            df.where(lit(v1) < 'id),
+            Array(sources.IsNotNull("id")))
+          checkPushedFilters(
+            format,
+            df.where(lit(v1) <= 'id),
+            Array(sources.IsNotNull("id")))
+          checkPushedFilters(
+            format,
+            df.where(lit(v1) === 'id),
+            Array(),
+            noScan = true)
+          checkPushedFilters(
+            format,
+            df.where(lit(v1) >= 'id),
+            Array(),
+            noScan = true)
+          checkPushedFilters(
+            format,
+            df.where(lit(v1) > 'id),
+            Array(),
+            noScan = true)
```

Comment:
Shall we minimize the above 20 lines into 5 lines like the following? The above looks too verbose unnecessarily.
```scala
checkPushedFilters(format, df.where(lit(v1) < 'id), Array(sources.IsNotNull("id")))
checkPushedFilters(format, df.where(lit(v1) <= 'id), Array(sources.IsNotNull("id")))
checkPushedFilters(format, df.where(lit(v1) === 'id), Array(), noScan = true)
checkPushedFilters(format, df.where(lit(v1) >= 'id), Array(), noScan = true)
checkPushedFilters(format, df.where(lit(v1) > 'id), Array(), noScan = true)
```

Reply:
For the other test cases, it's the same. We don't need to break lines when you make it one liner.